src/vs/workbench/contrib/debug/browser/media/debugHover.css

```diff
@@ -19,7 +19,7 @@
 	max-width: 700px;
 }
 
-.monaco-editor .debug-hover-widget .complex-value .title {
+.monaco-editor .debug-hover-widget .complex-value .title .tip {
```

Comment:
This is not good. Since you have made the previous rule more specific, thus it applies to elements that have a class `tip` as a child of `title`. However there are no such elements.
What I think you want is this rule to be aplied to both .title and .tip for that you have to do something like this

```
.monaco-editor .debug-hover-widget .complex-value .title, 
.monaco-editor .debug-hover-widget .complex-value .tip {
```